source/portable/posix/core_pkcs11_pal.c

```diff
@@ -115,42 +115,42 @@ static void prvLabelToFilenameHandle( const char * pcLabel,
 {
     if( ( pcLabel != NULL ) && ( pHandle != NULL ) && ( pcFileName != NULL ) )
     {
-        if( 0 == strncmp( pkcs11configLABEL_DEVICE_CERTIFICATE_FOR_TLS,
+        if( 0 == strncmp( &pkcs11configLABEL_DEVICE_CERTIFICATE_FOR_TLS,
                           pcLabel,
                           sizeof( pkcs11configLABEL_DEVICE_CERTIFICATE_FOR_TLS ) ) )
         {
             *pcFileName = pkcs11palFILE_NAME_CLIENT_CERTIFICATE;
             *pHandle = ( CK_OBJECT_HANDLE ) eAwsDeviceCertificate;
         }
-        else if( 0 == strncmp( pkcs11configLABEL_DEVICE_PRIVATE_KEY_FOR_TLS,
+        else if( 0 == strncmp( &pkcs11configLABEL_DEVICE_PRIVATE_KEY_FOR_TLS,
```

Comment:
Why is this needed?

Reply:
Added accidentally from my local workspace version, have reverted it.